test/lib/global_uid_test.rb

```diff
@@ -597,4 +591,17 @@ def with_modified_connections(increment:, servers:)
       yield
     end
   end
+
+  class TestErrorTracker
+    def initialize
+      @notifications = []
+    end
+
+    attr_accessor :notifications
+
+    def notify(exception)
+      ActiveRecord::Base.logger.error("GlobalUID error: #{exception.class} #{exception.message}")
```

Comment:
Is this line even needed?

Reply:
No, it should be extending the GlobalUid::ErrorTracker class and calling super instead, exercising the default handler.